content/guides/access-control.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Enable and configure access control
+weight: 1
+description: A tutorial for creating Role-based Access Control in Upbound
+---
+
+{{< hint "important" >}}
+This feature is in alpha and requires Spaces `v1.3.0`.
+{{< /hint >}}
+
+This guide introduces role-based access control (RBAC) in Upbound. RBAC allows you to apply privileges to your Upbound objects and control planes. Organizations using RBAC can manage user roles and privileges to determine the level of access a user can have within their role.
+
+RBAC allows organization administrators to:
+
+- build and assign custom roles
+- have a centralized set of policies to ensure appropriate access
+- enhance compliance workflows
+
+This guide highlights three RBAC workflows you might use in Upbound:
+
+- Kubernetes cluster management
+    -
+- Spaces/IAM
+    -
+
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+## Enable
+
+To enable RBAC in your organization, use the `--#TODO` feature flag:
```

Comment:
I guess we expect helm values to be the interface for people to use, like in https://docs.upbound.io/spaces/oidc/

Then, the user can enable hub authorization through `authorization.hubRBAC=true` and hub authentication through `authentication.hubIdentities=true` and structured authentication (using the file) through `authentication.structuredConfig=true`
ref: https://github.com/upbound/spaces/blob/8db6fec6b87e5a1a16f0799f92a58fd99b08d1d9/cluster/charts/spaces/values.yaml.tmpl#L147-L162